### PR TITLE
Navigation Bug

### LINKED ISSUE #74: Link Issues
## Expected Behavior

When the user is on the GamePage, and they click the myPoke or Pokedex, it would break because the Navigation component being rendered in the GamePage did not have access to the necessary props. 

## Actual/Current Behavior

When the user clicks the myPoke link or Pokedex link, the app crashes
## Steps to Reproduce the Problem
  1. Click earn coins link
  2.  Click Pokedex or myPoke link

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Fixed an issue where the app would crash when users clicked buttons on the navbar that was being rendered on the GamePage.
2. Removed the text-decoration on the links in the navigation bars. 
3. Saved type colors into scss variables and applied a shadow effect with those colors on mouse enter on the pokemon cards

## Purpose
This will enhance the user experience.

## Approach
Passed a boolean state value through the GamePage to the Navigation bar. Made a boolean state value to apply colors to the pokemon cards on mouse enter and leave. 

## Note
I did not mean for both of these changes to be submitted on this pull request. I made a mistake with working on different branches and work getting carried over to a new branch. Won't happen again!

Closes #74 , #75

### CODE REVIEW

File: pokeapp/src/components/PokemonCard.js [hunk 1]
```diff
@@ -13,6 +14,12 @@ const PokemonCard = ({
   );
   const dispatch = useDispatch();
 
+  const [isSelected, setIsSelected] = useState(false);
+
+  const applyTypeColor = (type) => {
+    return `card-${type}`;
+  }
+
```
Comment (gonzalespaulb) [inline pokeapp/src/components/PokemonCard.js:22]: Great job making this a dynamic function. Should save us a lot of extra lines

File: pokeapp/src/styling/components/_pokemon-card.scss [hunk 2]
```diff
@@ -13,6 +13,117 @@
         justify-content: space-between;
         align-items: baseline;
     }
+
```
Comment (gonzalespaulb) [inline pokeapp/src/styling/components/_pokemon-card.scss:16]: We can get rid of the background color, border radius, and padding by using this syntax. `&-fire, &-water, etc`. The only unique value for each selector would be the box shadow color.

**Review by gonzalespaulb — APPROVED** [review]
On the list for a refactor job other than that it looks good.

**Review by JayDampitan — APPROVED** [review]
Great job Kenny, appreciate all the hard work!

**Pull request merged**